src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1093,6 +1093,65 @@ def _ec_key_set_public_key_affine_coordinates(self, ctx, x, y):
 
         return ctx
 
+    def _private_bytes(self, encoding, format, encryption_algorithm,
+                       pem_write_func, evp_pkey, cdata):
+        if not isinstance(encoding, serialization.Encoding):
+            raise TypeError("encoding must be an item from the Encoding enum")
+
+        if not isinstance(format, serialization.Format):
+            raise TypeError("format must be an item from the Format enum")
+
+        # This is a temporary check until we land DER serialization.
+        if encoding is not serialization.Encoding.PEM:
+            raise ValueError("Only PEM encoding is supported by this backend")
+
+        if format is serialization.Format.PKCS8:
+            write_bio = self._lib.PEM_write_bio_PKCS8PrivateKey
+            key = evp_pkey
+        elif format is serialization.Format.TraditionalOpenSSL:
+            write_bio = pem_write_func
```

Comment:
`pem_write_func` isn't a great name for what this is, it's not like PKCS8 isn't pem.